lib/entities/request/ClassicRequest.ts

```diff
@@ -23,6 +23,7 @@ export interface ClassicConfig {
   forceMixedRoutes?: boolean;
   debugRoutingConfig?: string;
   unicornSecret?: string;
+  quoteSpeed?: string;
```

Comment:
this should be an enum, no?

Reply:
In general I def agree. In this case as you mentioned, we are trying to jam.. which means we might need to add more variant of quoteSpeed. Once we solidify the final design, we can revisit to make it a enum.

created ticket for tracking - [ROUTE-79](https://linear.app/uniswap/issue/ROUTE-79/quick-quotespeed-follow-up-make-quotespeed-enum-in-both-ura-and)